src/server/src/utils/getUniqueId.ts

```diff
@@ -0,0 +1,10 @@
+import Game from '../classes/Game';
+import { v1 as uuid } from 'uuid';
+
+export const getUniqueId = (games: Map<string, Game>): string => {
+  const id: string = uuid().substr(0, 4);
+  if (games.has(id)) {
+    getUniqueId(games);
```

Comment:
seems like it could generate an unnecessary amount of calls, if something goes wrong.
why not do:
```
const id: string;
do {
  id = uuid().substr(0, 4);
} while (games.has(id));
return id;
```